src/Greetingandintro/Greeting.jsx

```
@@ -0,0 +1,749 @@
+        setTimeout(() => {
+          playAudio(audioItem.src, `${lesson.id}-${audioIndex}`);
+        }, 500);
+      }
+    }
+  }, [currentLessonIdx, currentExIdx]);
+
+  // Persist completed lessons
+  useEffect(() => {
+    try {
+      localStorage.setItem("greetings_completed", JSON.stringify(completedLessons));
+    } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)